src/web/components/Core/ErrorView.tsx

```diff
@@ -0,0 +1,23 @@
+import { analyticsIdentifier } from '../../utils/errorHelpers';
+
+type ErrorViewProps = {
```

Comment:
I extracted the props to a type here, but otherwise it is a cut-and-paste from `ErrorView.tsx`